.github/workflows/tests.yaml

```diff
@@ -164,8 +164,15 @@ jobs:
           go install sigs.k8s.io/kind@v0.20.0
           kind create cluster
 
```

Comment:
Can we remove Kind?

Reply:
There's a test that specifically uses the k8s api.

 - https://github.com/sigstore/cosign/blob/d6c40c940fb04cd62156a69b32f46df5a1012bfa/test/e2e_test.go#L2014

Reply:
Added the comment. What do you mean by splitting into its own test?

Reply:
Sorry, I meant a comment in the action for why `kind` is needed.

Since `kind` is only needed for one test, it could be in its own e2e test so that it can run in parallel with other tests and improve overall runtime. We can do that later on, there's a few places we can clean up testing in cosign.

Reply:
I think later. the "kind" step is always ~10 seconds.